### DISCUSSION TITLE
[Feat]: 推送与截图相关功能使用问题

### DISCUSSION THREAD

**Discussion by MusherM** [moesnow/March7thAssistant#116] [Q&A]
### 请确保您已阅读以上注意事项，并勾选下方的确认框。

- [X] 我确定这是一个从未被提出和实现过的功能。
- [X] 我已经在 [Issue Tracker](https://github.com/moesnow/March7thAssistant/issues) 以及 [Pull Request](https://github.com/moesnow/March7thAssistant/pulls) 中找过我要提出的功能，没有找到相同功能的ISSUE和PR。
- [X] 我已知晓并同意，此处仅用于建议新功能。若这个 Issue 是关于其他非主题的问题，则我的 Issue 可能会被无条件自动关闭/锁定。
- [X] 我已知晓并同意，如果我不按照模板提供的指示进行填写，则我的 Issue 可能会被无条件删除。

### 需要添加的功能内容

我在文档中看到telegram方式支持截图，但我该在哪里开启截图功能？截图到底会截什么？不论在app图形界面还是在config文件中都没有看到相关选项，文档中也没有相关说明

**Comment by moesnow** [comment]
参考 `README.md` 中 `界面展示` 里手机截图的内容，支持截图的推送方式会默认开启，如果不支持就只有文字。

**Reply by MusherM** [reply]
好的 感谢解答
